tests/controls/gridtest.cpp

```diff
@@ -716,6 +710,9 @@ TEST_CASE_METHOD(GridTestCase, "Grid::SelectUsingEndKey", "[grid]")
 
     sim.KeyDown(WXK_END, wxMOD_CONTROL | wxMOD_SHIFT);
     sim.KeyUp(WXK_END, wxMOD_CONTROL | wxMOD_SHIFT);
+#ifdef __WXGTK20__
+    wxMilliSleep(10);
+#endif
```

Comment:
Looks like this could be removed too if we just used extra 10ms for all emulated key presses?